eZ/Publish/API/Repository/Exceptions/ContentFieldValidationException.php

```diff
@@ -13,7 +13,7 @@
 /**
  * This Exception is thrown on create or update content one or more given fields are not valid.
  */
-abstract class ContentFieldValidationException extends ForbiddenException
+abstract class ContentFieldValidationException extends ContentValidationException
```

Comment:
@andrerom But then we're back to the original BC break: That certain errors ("required field is missing") which used to throw ContentValidationException now throw ContentFieldValidationException. If there are BC breaks either way, we need to decide 1) which is worse, and 2) can we do the change before 7.0?